Remotion/Core/Development.Analyzers.IntegrationTests/CSharpAnalyzerVerifier.cs

```diff
@@ -75,8 +72,6 @@ private static ReferenceAssemblies GetReferenceAssemblies (Assembly assembly)
     {
       return assembly.GetCustomAttribute<TargetFrameworkAttribute>()!.FrameworkName switch
       {
-          ".NETCoreApp,Version=v6.0" => ReferenceAssemblies.Net.Net60,
-          ".NETCoreApp,Version=v7.0" => s_net70.Value,
           ".NETCoreApp,Version=v8.0" => s_net80.Value,
```

Comment:
THis is where I would like to keep the hint on using `ReferenceAssemblies.Net.Net80,` once it is available.

Reply:
Idk why, but I thought that this hint belonged in the Jira ticket. I'll leave it as a comment on Jira and add it here as well.